toolchain/lower/file_context.cpp

```diff
@@ -463,4 +479,17 @@ auto FileContext::BuildType(SemIR::InstId inst_id) -> llvm::Type* {
   }
 }
 
+auto FileContext::BuildGlobalVariableDecl(SemIR::InstId inst_id)
+    -> llvm::GlobalVariable* {
+  auto var_storage = sem_ir().insts().Get(inst_id).As<SemIR::VarStorage>();
+  auto mangled_name =
```

Comment:
```suggestion
  // TODO: Mangle name.
  auto mangled_name =
```

I'm not exactly sure what the samantics for `GetAsStringIfIdentifier` will be once name mangling is actually added, but seems worth leaving a breadcrumb here.

Reply:
Done.